### DISCUSSION TITLE
[@types/mailparser] imports nonexistent type from iconv-lite

### DISCUSSION THREAD

**Discussion by crossan007** [DefinitelyTyped/DefinitelyTyped#67073] [Issues with a @types package]
### Summary
The `@types/mailparser` package includes this import:
```
import { DecoderStream } from "iconv-lite";
```

`DecoderStream` is not defined in the `iconv-lite` package, and causes a `tsc` error `TS2724` in `'node_modules/@types/mailparser/index.d.ts`: `'"iconv-lite"' has no exported member named 'DecoderStream'. Did you mean 'decodeStream'?`

### Related Issues
* https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56983
* https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11735
* https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22428

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
this symbol seems to be defined in `iconv-lite` types:
https://unpkg.com/browse/iconv-lite@0.6.3/lib/index.d.ts
It should be visible in package installation files

**Reply by crossan007** [reply]
I see that _now_ 🤔   

Maybe there's something wrong on my local machine that's causing this error.

When I figure it out, I'll update this issue.
